config/code_schools.yml

```diff
@@ -438,6 +438,13 @@
   va_accepted: false
   full_time: true
   hardware_included: false
-  
-    
-  
+
+- name: Codeup
+  url: http://codeup.com/
+  address1: 600 Navarro St., #350
```

Comment:
Does this show up on the page or is it ignored as a comment? You may have to put this in quotes.